packages/node/src/configure/SubqueryProject.ts

```diff
@@ -100,16 +101,29 @@ export class SubqueryProject implements ISubqueryProject {
       NOT_SUPPORT('<1.0.0');
     }
 
+    const fileCacheDir = await getFileCacheDir(reader, root);
+
     return loadProjectFromManifestBase(
       manifest.asV1_0_0,
       reader,
       path,
       root,
       networkOverrides,
+      fileCacheDir,
     );
   }
 }
 
+async function getFileCacheDir(
+  reader: Reader,
+  projectRoot: string,
+): Promise<string> {
+  if (isTmpDir(projectRoot)) return projectRoot;
+  if (reader instanceof LocalReader) return makeTempDir();
```

Comment:
This is going to create a new temp dir for every worker if the project reader is a local reader, it needs to parse the directory to other workers